config/agoradev/develop/agora-bastian.yaml

```diff
@@ -12,7 +12,7 @@ parameters:
   # Name of an the environment either develop, staging or prod
   Environment: "develop"
   # Name of an existing VPC subnet to run the instance in
-  VpcSubnet: "PublicSubnet"
+  VpcSubnet: "PrivateSubnet"
```

Comment:
Note: AWS will create a new EC2 instance on this change which means that the Github self-hosted runners will need to be manually re-configured.